From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Tue, 22 Sep 2020 18:09:57 +0000
Subject: [PATCH] Use stateless gdmconfdir for integration into Clear Linux

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 7e336bf..f48f206 100644
--- a/meson.build
+++ b/meson.build
@@ -15,7 +15,7 @@ cc = meson.get_compiler('c')
 # Options
 gdm_prefix = get_option('prefix')
 
-gdmconfdir = (get_option('sysconfsubdir') == '')? gdm_prefix / get_option('sysconfdir') : gdm_prefix / get_option('sysconfdir') / get_option('sysconfsubdir')
+gdmconfdir = gdm_prefix / get_option('datadir') / 'gdm'
 dmconfdir = (get_option('dmconfdir') != '')? get_option('dmconfdir') : gdm_prefix / get_option('sysconfdir') / 'dm'
 udev_dir = get_option('udev-dir')
 at_spi_registryd_dir = (get_option('at-spi-registryd-dir') != '')? get_option('at-spi-registryd-dir') : gdm_prefix / get_option('libexecdir')
