From cf7a76562cc7729b38446388948d4c0b402ad02a Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Wed, 29 Nov 2017 14:15:56 -0600
Subject: [PATCH] conflict with gdm-hw-accel.service

---
 data/gdm.service.in | 15 ++-------------
 1 file changed, 2 insertions(+), 13 deletions(-)

diff --git a/data/gdm.service.in b/data/gdm.service.in
index 72201c1..71e696a 100644
--- a/data/gdm.service.in
+++ b/data/gdm.service.in
@@ -5,19 +5,8 @@ Description=GNOME Display Manager
 Conflicts=getty@tty@GDM_INITIAL_VT@.service
 After=getty@tty@GDM_INITIAL_VT@.service
 
-# replaces plymouth-quit since it quits plymouth on its own
-Conflicts=@PLYMOUTH_QUIT_SERVICE@
-After=@PLYMOUTH_QUIT_SERVICE@
-
-# Needs all the dependencies of the services it's replacing
-# pulled from getty@.service and @PLYMOUTH_QUIT_SERVICE@
-# (except for plymouth-quit-wait.service since it waits until
-# plymouth is quit, which we do)
-After=rc-local.service plymouth-start.service systemd-user-sessions.service
-
-# GDM takes responsibility for stopping plymouth, so if it fails
-# for any reason, make sure plymouth still stops
-OnFailure=plymouth-quit.service
+Conflicts=gdm-hw-accel.service
+After=rc-local.service systemd-user-sessions.service gdm-disable-a2dp-pulseaudio.service
 
 [Service]
 ExecStart=@sbindir@/gdm
-- 
2.15.0

